src/libsyntax_pos/edition.rs

```diff
@@ -37,6 +38,16 @@ impl fmt::Display for Edition {
 }
 
 impl Edition {
+    pub fn from_session() -> Edition {
+        GLOBALS.with(|globals| globals.edition.load())
+    }
+
+    // FIXME: Fix rustdoc's global usage and remove this function
+    // and make the `edition` field immutable.
```

Comment:
@Zoxc, do you plan to address this FIXME in this PR?

Reply:
No.